fact-bounty-client/src/shared/components/TopNavBar.js

```diff
@@ -1,75 +1,117 @@
-import React, {Component}   from 'react'
-import Button               from '@material-ui/core/Button'
-import AppBar               from '@material-ui/core/AppBar'
-import Toolbar              from '@material-ui/core/Toolbar'
-import Typography           from '@material-ui/core/Typography'
-import {ExpandMore, Person} from '@material-ui/icons'
-import {Menu, MenuItem}     from '@material-ui/core'
-import {Link}               from 'react-router-dom'
+import React, { Component } from "react";
+import Button from "@material-ui/core/Button";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import { ExpandMore, Person } from "@material-ui/icons";
+import { Menu, MenuItem } from "@material-ui/core";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+
+import { logoutUser } from "../../pages/dashboard/actions/dashboardActions";
 
 const styles = {
-	navbar: {
-		backgroundColor: '#fafafa',
-		zIndex: '15'
-	},
-	navbarTitle: {
-		flexGrow: 2,
-		MozUserSelect: 'none',
-		WebkitUserSelect: 'none',
-		msUserSelect: 'none',
-		textDecoration: 'none'
-	},
-	navbarLinks: {
-		letterSpacing: '1px',
-		fontWeight: '500',
-		marginLeft: '24px'
-	},
-	link: {
-		textDecoration: 'none'
-	}
-}
+  navbar: {
+    backgroundColor: "#fafafa",
+    zIndex: "15"
+  },
+  navbarTitle: {
+    flexGrow: 2,
+    MozUserSelect: "none",
+    WebkitUserSelect: "none",
+    msUserSelect: "none",
+    textDecoration: "none"
+  },
+  navbarLinks: {
+    letterSpacing: "1px",
+    fontWeight: "500",
+    marginLeft: "24px"
+  },
+  link: {
+    textDecoration: "none"
+  }
+};
 
 class TopNavBar extends Component {
+  onLogoutClick = e => {
+    e.preventDefault();
+    this.props.logoutUser();
+  };
+
+  state = {
+    anchorEl: null
+  };
 
-	state = {
-		anchorEl: null
-	}
+  handleToggle = event => {
+    this.setState({ anchorEl: event.currentTarget });
+  };
 
-	handleToggle = event => {
-		this.setState({anchorEl: event.currentTarget})
-	}
+  handleClose = () => {
+    this.setState(state => ({
+      anchorEl: null
+    }));
+  };
 
-	handleClose = () => {
-		this.setState(state => ({
-			anchorEl: null
-		}))
-	}
+  render() {
+    const { isAuthenticated} = this.props.auth;
+    const { anchorEl } = this.state;
```

Comment:
This variable is not used. Why is it here ?

Reply:
> This variable is not used. Why is it here ?

{anchorEl} was already present in code base, but I will remove it now